tests/test_ops.py

```diff
@@ -30,17 +30,17 @@ class TestOPSExpression(object):
         ('Eq(u,3*a - 4**a)', 'void OPS_Kernel_0(ACC<float> & ut01)\n'
          '{\n  ut01(0) = -2.97015324253729F;\n}'),
         ('Eq(u, u.dxl)',
-         'void OPS_Kernel_0(ACC<float> & ut01, const float *h_x)\n'
-         '{\n  float r0 = 1.0/*h_x;\n  '
-         'ut01(0) = (-2.0F*ut01(-1) + 5.0e-1F*ut01(-2) + 1.5F*ut01(0))*r0;\n}'),
-        ('Eq(v,1)', 'void OPS_Kernel_0(ACC<float> & vt01)\n'
-         '{\n  vt01(0, 0) = 1;\n}'),
+         'void OPS_Kernel_0(ACC<float> & ut00, const float *h_x)\n'
```

Comment:
Why the changes in this test?
The changes from `ut00` to `ut01`was done when applying Maelso's fix when a negative time dimension is acessed in the iet.

Reply:
Hummm i have no idea, will undo the changes.